addon/templates/components/tag-input.hbs

```diff
@@ -8,10 +8,11 @@
 {{~/each~}}
 
 <li class="emberTagInput-new">
-  {{input
-    disabled=readOnly
-    class=(concat 'emberTagInput-input js-ember-tag-input-new' (if readOnly ' is-disabled'))
-    maxlength=maxlength
-    placeholder=placeholder
-  }}
+  <Input
+    @disabled={{this.readOnly}}
+    class={{concat 'emberTagInput-input js-ember-tag-input-new' (if readOnly ' is-disabled')}}
+    maxlength={{this.maxlength}}
+    placeholder={{this.placeholder}}
+    aria-label={{this.ariaLabel}}
+  />
```

Comment:
Based on my googling, I think the switch to angle brackets will break the addon for anyone using Ember < 3.10 (which I don't know if anyone is). Could you add the aria label without switching to angle brackets? Otherwise I will have to release this change as a major version bump

Reply:
Unfortunately, no, I've not found a way to do it with curly brace invocation. And yeah, I'm pretty sure you're correct that it'll break older versions of Ember.

Reply:
What about using a plain html `<input>` tag? I can't remember if there's a reason I used `{{input}}`

Reply:
Yes, the HTML input tag will work if you're not relying on anything from the ember helper like a live update.

Reply:
You interested in giving it a shot? If not, I'll probably give it a try after merging this before I cut a release

Reply:
Done. Looks like it was easy enough.